src/lib/components/search/search.svelte.ts

```
@@ -5,3 +5,39 @@ export const searchState = $state({
     isResultsShown: false,
     isResultsMinimized: false,
 });
+
+export function buildSearchUrl(params: {query: string; lat: string; lng: string}): string {
+    const sp = new URLSearchParams();
+    if (params.query) sp.set('q', params.query);
```

ESLint

[error] Expected { after 'if' condition.

(curly)